group.html

```diff
@@ -88,6 +88,20 @@ <h3>Ryan Morris
 												<li><a href="http://ryanramorris.space" class="button">Website</a></li>
 											</ul>
 										</article>
+										<article>
+											<a href="#" class="image"><img src="images/aj_photo_1.jpg.jpg" alt="" /></a>

```

Comment:
Do you mean to have the filename be *.jpg.jpg?

Reply:
I did not.